services/fileuploader/fileuploader.go

```diff
@@ -43,6 +48,17 @@ func NewStaticProvider(storageSettings map[string]StorageSettings) Provider {
 		init:            make(chan struct{}),
 		storageSettings: storageSettings,
 	}
+	s.fileManagerMap = make(map[string]func() (filemanager.FileManager, error))
+	for workspaceID, settings := range storageSettings {
+		if settings.Bucket.Type != "" {
```

Comment:
Would it make sense to return an error in this case?

Reply:
It would, but then we'd have to change function signatures in multiple places imo.
If an error weren't returned, subsequent call to get a file manager would fail with an error in any case.